overlord/hookstate/ctlcmd/mount.go

```diff
@@ -68,7 +68,7 @@ func matchMountPathAttribute(path string, attribute interface{}, snapInfo *snap.
 
 	expandedPattern := snapInfo.ExpandSnapVariables(pattern)
 
-	pp, err := utils.NewPathPattern(expandedPattern)
+	pp, err := utils.NewPathPattern(expandedPattern, false)
```

Comment:
shouldn't this be consistent with what we do in mount_control.go?

Reply:
Yes, I since changed this. Thanks!